contrib/sarplus/python/setup.py

```diff
@@ -14,11 +19,21 @@ def __str__(self):
         return pybind11.get_include(self.user)
 
 
+DEPENDENCIES = [
+    "numpy",
+    "pandas",
+    "pyarrow>=1.0.0",
+    "pybind11>=2.2",
+    "pyspark>=3.0.0"
+]
+
 setup(
     name="pysarplus",
-    version="0.2.6",
+    version=(Path(__file__).resolve().parent / "VERSION").read_text().strip(),
     description="SAR prediction for use with PySpark",
-    url="https://github.com/Microsoft/Recommenders/contrib/sarplus",
+    long_description=(Path(__file__).resolve().parent / "README.md").read_text(),
+    long_description_content_type='text/markdown',
+    url="https://github.com/microsoft/recommenders/tree/main/contrib/sarplus",
     author="Markus Cozowicz",
     author_email="marcozo@microsoft.com",

```

Comment:
Same as before, I would change the author and author email to be consistent with the rest of the repo https://github.com/microsoft/recommenders/blob/main/recommenders/__init__.py#L6 and reflect current and future contributions

Reply:
Use RecoDev instead